api/models.py

```
@@ -53,6 +53,29 @@ def __str__(self) -> str:
+class ItemImage(models.Model):
+    item = models.ForeignKey(
+        Item,
+        on_delete=models.CASCADE,
+        related_name="images",
+    )
+    image_url = models.URLField(blank=True)
+    image_file = models.ImageField(upload_to="item_images/", blank=True, null=True)
+    position = models.PositiveSmallIntegerField(default=0)
+
+    class Meta:
+        ordering = ["position", "id"]
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)